src/components/moderation/ReportDialog/utils/useReportOptions.ts

```diff
@@ -1,121 +1,321 @@
+        },
+        {
+          title: _(msg`Glorification of Violence`),
+          reason: 'tools.ozone.report.defs#reasonViolenceGlorification',
+        },
+        {
+          title: _(msg`Extermist Content`),
+          reason: 'tools.ozone.report.defs#reasonViolenceExtremistContent',
+        },
+        {
+          title: _(msg`Human Trafficking`),
+          reason: 'tools.ozone.report.defs#reasonViolenceTrafficking',
+        },
+        {
+          title: _(msg`Other Violent Content`),
+          reason: 'tools.ozone.report.defs#reasonViolenceOther',
         },
-        ...common,
       ],
-      chatMessage: [
+    },
+    sexualAdultContent: {
+      key: 'sexualAdultContent',
+      title: _(msg`Sexaul and Adult Content`),
+      description: _(msg`Adult, child or animal sexual abuse`),
+      sort: 3,
+      options: [
+        {
+          title: _(msg`Adult Sexual Abuse Content`),
+          reason: 'tools.ozone.report.defs#reasonSexualAbuseContent',
+        },
+        {
+          title: _(msg`Non-Consensual Intimate Imagery`),
+          reason: 'tools.ozone.report.defs#reasonSexualNCII',
+        },
+        {
+          title: _(msg`SexualSextortion`),
+          reason: 'tools.ozone.report.defs#reasonSexualSextortion',
+        },
+        {
+          title: _(msg`Deepfake Adult Content`),
+          reason: 'tools.ozone.report.defs#reasonSexualDeepfake',
+        },
+        {
+          title: _(msg`Animal Sexual Abuse`),
+          reason: 'tools.ozone.report.defs#reasonSexualAnimal',
+        },
+        {
+          title: _(msg`Unlabelled Adult Content`),
+          reason: 'tools.ozone.report.defs#reasonSexualUnlabeled',
+        },
+        {
+          title: _(msg`Other Sexual Violence Content`),
+          reason: 'tools.ozone.report.defs#reasonSexualOther',
+        },
+      ],
+    },
+    harassmentHate: {
+      key: 'harassmentHate',
+      title: _(msg`Harassment or Hate`),
+      description: _(
+        msg`Targeted attacks, hate speech, or organized harassment`,
+      ),
+      sort: 4,
+      options: [
+        {
+          title: _(msg`Trolling`),
+          reason: 'tools.ozone.report.defs#reasonHarassmentTroll',
+        },
+        {
+          title: _(msg`Targeted Harassment`),
+          reason: 'tools.ozone.report.defs#reasonHarassmentTargeted',
+        },
+        {
+          title: _(msg`Hate Speech`),
+          reason: 'tools.ozone.report.defs#reasonHarassmentHateSpeech',
+        },
+        {
+          title: _(msg`Doxxing`),
+          reason: 'tools.ozone.report.defs#reasonHarassmentDoxxing',
+        },
+        {
+          title: _(msg`Other Harassing or Hateful Content`),
+          reason: 'tools.ozone.report.defs#reasonHarassmentOther',
+        },
+      ],
+    },
+    misleading: {
+      key: 'misleading',
+      title: _(msg`Misleading`),
+      description: _(msg`Spam, scams, false info or impersonation`),
```

Comment:
```suggestion
      description: _(msg`Spam, scams, false info, or impersonation`),
```
nit: Elsewhere, Oxford commas are used consistently, so I think there should be one here too